src/App.css

```diff
@@ -0,0 +1,280 @@
+  padding-right: 1.7rem;
+  display: inline-block;
+  font-weight: 700;
+  -webkit-appearance: button;
+  cursor: pointer;
+  color: #1a1a1b;
+  background-color: transparent;
+  border-color: transparent;
+}
+
+button:focus, .search:focus, .form-styled input:focus, .form-styled textarea:focus {
+   outline: none
+}
+
+button:hover {
+  color: var(--secondary-color);
+}
+
+button::after {
+  font-family: "Font Awesome 5 Free"; 
+  position: absolute;
+  right: .5rem;
+  -webkit-transition: all .2s ease-in-out;
+  transition: all .2s ease-in-out;
+  text-transform: none;
+}
+
+button:hover::after {
+  color: var(--secondary-color);
+  right: .3rem;
+}
+
+.btn-complete::after {
+  content: '\f00c';
+}
+
+.btn-uncomplete::after {
+  content: '\f00d';
+}
+
+.btn-remove::after {
+  content: '\f2ed';
+}
+
+/* 
+  Main styling
+*/
+.container {
+  display: grid;
+  height: 100%;
+  grid-template-areas: "content sidebar"
+                      "footer  footer";
+  grid-template-columns: calc(70% - var(--gap_vertical) / 2) calc(30% - var(--gap_vertical) / 2);
+  grid-template-rows: 1fr auto;
+  grid-gap: var(--gap_vertical);
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 0 1em;
+}
+
+/* Search form section */
+form, .form {    
+  display: flex;
+}
+
+.form-search {
+  grid-column: 1 / 4;
+  margin-top: 2rem;
+  justify-content: center;
+}
+
+.search {
+  border: 0;
+  margin-left: .5rem;
+  border-bottom: 1px solid black;
+  background: transparent;
+}
+
+.search::before {
+  font-family: "Font Awesome 5 Free"; 
+  margin-right: .5em;
+  content: '\f002';
+  -webkit-font-smoothing: antialiased;
+}
+
+/* Todo items section */
+.card-container {
+  grid-area: content;
+  display: grid;
+  grid-auto-rows: min-content;
+  grid-gap: var(--gap_vertical);
+  grid-template-columns: repeat(3,1fr);
+  margin-bottom: var(--gap_gorizontal);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  box-shadow: var(--element-shadow);
+  background: white linear-gradient(135deg, var(--secondary-color__light) 10px, rgb(255, 255, 255) 0);
+  -webkit-transition: all .2s ease-in-out;
+  transition: all .2s ease-in-out;
+}
+
+.card.completed {
+   background: rgba(255, 255, 255, 0.6);
+   box-shadow: 0.25em 0.25em 0.5em rgba(0,0,0,.0);
+}
+
+.completed .card-body {
+  text-decoration: line-through;
+}
+
+.card-author {
+  color: #898d93;
+  margin-bottom: .5rem;
+}
+
+.date::before {
+  margin-right: .5rem;
+  content: '\2012';
+}
+
+.card-body {
+  margin-bottom: .75rem;
+}
+
+/* Sidebar section */
+aside {
+  grid-area: sidebar;
+  align-self: center;
+  display: flex;
+  align-self: center;
+  justify-content: center;
+  padding: 10px;
+  border-left: 1px dotted lightgrey;
+}
+
+/* New Todo Item Form*/
+.form-styled {
+  flex-direction: column;
+  background-color: rgba(255, 255, 255, 0.6);
+  flex: 0 1 17rem;
+  padding: 2rem 1.5rem;
+  box-shadow: 0.25em 0.25em 0.5em rgba(0,0,0,.0);
+  -webkit-transition: all .2s ease-in-out;
```

Comment:
no need to write prefixed values manually as there is [autoprefixer enabled](https://github.com/andrewgaryga/Corporate-task-manager--React-/pull/1/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R23)